code/datums/components/food/edible.dm

```diff
@@ -319,6 +319,8 @@ Behavior that's still missing from this component that original food items had t
 
 
 	else //If you're feeding it to someone else.
+		if(eater.surgeries.len)
+			return
```

Comment:
People always forget to close their surgeries. This should also require them to be lying down and those surgeries to be mouth surgeries.
Also, organs themselves have the behaviour to prevent surgery feeding specifically on them. That should be fixed instead, or removed if the behaviour is now global.

Reply:
> People always forget to close their surgeries. This should also require them to be lying down and those surgeries to be mouth surgeries.

Fair enough.



> Also, organs themselves have the behaviour to prevent surgery feeding specifically on them.

No. IIRC they have behavior that handles them not accidentally getting a *bite* taken out of them during surgery. That works fine.

All this does is prevent the progress bar from starting due to the edible component. There is no actual problem of organs being eaten, that was fixed.

Reply:
It should never get to this point for organs in the first place. pre_eat is called inside of CanConsume which runs before this part of the proc. If something is an organ, then it won't be able to be consumed if there is a valid surgery in the location that you are trying to feed. 

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/f3fffe0c-429e-40fa-81e0-0719e82416fd)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/cdadba2c-6155-4be3-92b5-41f7534a1e8a)

Reply:
This is most likely an issue with simplified bodyzone targeting